docs/kratos/passwordless/05_passkeys.mdx

```diff
@@ -7,7 +7,7 @@ sidebar_label: Passkeys
 # Passkeys
 
 [Passkeys](https://fidoalliance.org/passkeys/) are, as described by the [FIDO Alliance](https://fidoalliance.org/), "a password
-replacement that provide faster, easier, and more secure sign-ins to websites and apps across a user's devices."
+replacement that provides faster, easier, and more secure sign-ins to websites and apps across a user's devices."
```

Comment:
In my defense, this is copied directly from the FIDO landing page. :D

<img width="959" alt="image" src="https://user-images.githubusercontent.com/29116492/204472266-2b517b5d-1ac9-450d-93ad-377dc9b74efb.png">